### PR TITLE
MGDSTRM-5075: Updated left nav for App Services bundle on console.redhat.com

### PR DESCRIPTION
**Pull request by srijitamukherjee**
Fixes: https://issues.redhat.com/browse/MGDSTRM-5075
Corresponding PR: https://gitlab.cee.redhat.com/mk-ci-cd/application-services-ui/-/merge_requests/154

xyz

Signed-off-by: Srijita Mukherjee <srmukher@redhat.com>

### CODE REVIEW

File: chrome/application-services-navigation.json [hunk 1]
```diff
@@ -9,20 +9,57 @@
...
+            "title": "Service Registry",
             "expandable": true,
             "routes": [
                 {
                     "appId": "applicationServices",
-                    "title": "Kafka Instances",
+                    "title": "Service Registry Instances",
                     "isBeta": true,
-                    "href": "/application-services/streams/kafkas"
+                    "href": "/application-services/sr"
```
Comment (pmuir) [inline chrome/application-services-navigation.json]: I think the desired URL is `/application-services/service-registry`

Comment (srijitamukherjee) [reply chrome/application-services-navigation.json]: I will change it.

File: chrome/application-services-navigation.json [hunk 2]
```diff
@@ -9,20 +9,51 @@
...
+                    "title": "Service Registry Instances",
                     "isBeta": true,
-                    "href": "/application-services/streams/kafkas"
-                },
+                    "href": "/application-services/service-registry"
+                },              
+                {
+                    "title": "Documentation",
+                    "isExternal": true,
+                    "href": "https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka"
```
Comment (jgiardino) [inline chrome/application-services-navigation.json]: This is the URL for Kafka documentation. There's a separate URL for Service Registry documentation:

https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry

Comment (srijitamukherjee) [reply chrome/application-services-navigation.json]: Thanks :), I have updated the URL.

File: main.yml [hunk 3]
```diff
@@ -696,14 +699,22 @@ streams:
...
 
+service-registry:
+  title: "Service Registry"
+  frontend:
+    sub_apps:
+      - id: sr
+        title: "Service Registry"
+      - id: streams-docs
+        title: Documentation
+        navigate: https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/
```
Comment (jgiardino) [inline main.yml]: This should be updated to provide documentation for Service Registry. It still shows the URL and id for Kafka.

Comment (jgiardino) [reply main.yml]: BTW, about the 404, according to the CCS team, the contents are not live yet, but this should be the correct URL: 
https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry

Comment (srijitamukherjee) [reply main.yml]: I have updated!
Thanks for the review

**Review by pmuir — APPROVED** [review]

**Comment by pmuir** [comment]
We'll merge the router change first then this

File: main.yml [hunk 4]
```diff
@@ -696,14 +699,22 @@ streams:
...
         title: "Resources"
       - id: streams-docs
         title: Documentation
         navigate: https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/
 
+service-registry:
+  title: "Service Registry"
+  frontend:
+    sub_apps:
+      - id: sr
```
Comment (ajaypratap003) [inline main.yml]:
```suggestion
      - id: service-registry
```

I think id should be `service-registry` instead 
@pmuir Please correct me if I'm wrong.

File: main.yml [hunk 5]
```diff
@@ -696,14 +699,22 @@ streams:
...
+        title: "Service Registry"
```
Comment (ajaypratap003) [inline main.yml]:
```suggestion
        title: "Instances"
```
Title  should be `Instances` according to  design doc.  https://marvelapp.com/prototype/5dhc6c0/screen/81843300
@jgiardino ^^

File: main.yml (hunk 4 above)
Comment (pmuir) [reply main.yml]: Not clear tbh. Doing what @ajaypratap003 suggests will result in `application-services/service-registry/service-registry which is almost certainly wrong.

File: main.yml (hunk 5 above)
Comment (pmuir) [reply main.yml]: This is also inconsistent with Kafka...

File: main.yml (hunk 4 above)
Comment (pmuir) [reply main.yml]: This is a good point though @srijitamukherjee - I'm not sure that you put the nested routing properly into app-services -ui

Comment (ajaypratap003) [reply main.yml]: > This is a good point though @srijitamukherjee - I'm not sure that you put the nested routing properly into app-services -ui

@pmuir I can't see nested route in app-services-ui for service registry. We need to fix nested route based on what we want to keep id for SR. Can we have id  `registries` similar to kafkas

File: main.yml (hunk 5 above)
Comment (ajaypratap003) [reply main.yml]: > This is also inconsistent with Kafka...

I think we should have `Instances` for both kafka ad SR

Comment (pmuir) [reply main.yml]: Tiffany updated the design to "Service Registry Instances"

File: main.yml (hunk 4 above)
Comment (pmuir) [reply main.yml]: I spoke to Tiffany - let's go with `instances` as the id

File: main.yml (hunk 5 above)
Comment (srijitamukherjee) [reply main.yml]: **https://prod.foo.redhat.com:1337/beta/application-services/overview**
Screenshot from 2021-09-16 19-06-49

**https://prod.foo.redhat.com:1337/beta/application-services/service-accounts**
Screenshot from 2021-09-16 19-07-28

**https://prod.foo.redhat.com:1337/beta/application-services/service-registry**
Screenshot from 2021-09-16 19-07-53


@pmuir @ajaypratap003 This is how the UI looks now, after the requested changes. Please let me know if there is any other change required.
Still not sure how everything integrates together!!

**Comment by Hyperkid123** [comment]
@pmuir just ping me whenever you feel like it's ready to be merged.

**Review by pmuir — APPROVED** [review]

**Comment by pmuir** [comment]
@Hyperkid123 ready for merge

**Pull request merged**